packages/lexical-devtools/src/panel/components/InspectedElementProp/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import './index.css';
+
+import {NodeProperty} from 'packages/lexical-devtools/types';
+import * as React from 'react';
+
+function InspectedElementProp({
```

Comment:
Maybe a different component name to avoid any confusion with the React props.

```suggestion
function InspectedElementRow({
```